doc/modules/linear_model.rst

```diff
@@ -1148,6 +1148,72 @@ Also, this estimator is different from the R implementation of Robust Regression
 squares implementation with weights given to each sample on the basis of how much the residual is
 greater than a certain threshold.
 
+.. _quantile_regression:
+
+Quantile Regression
+===================
+
+Quantile regression estimates median or other quantiles of :math:`y` conditional on :math:`X`,
+    while OLS estimates conditional mean.
```

Comment:
Please remove spaces in the beginning of the line, see rendered docs at https://20523-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/modules/linear_model.html#quantile-regression